pkg/registry/registry.go

```diff
@@ -0,0 +1,63 @@
+package registry
+
+import (
+	"encoding/json"
+	"gz-beat-shipper/pkg/filehandler"
+	"log"
+)
+
+type Registry struct {
+	ParsedFiles []File `json:"ParsedFile"`
+}
+
+type File struct {
+	Name string `json:"name"`
+}
+
+// Append File struct to Registry Struct with ParsedFiles slice
+func (r *Registry) AppendFileToRegistry(f File) {
+	r.ParsedFiles = append(r.ParsedFiles, f)
+}
+
+// Convert Registry Struct to bytes in JSON
+func ConvertRegistryToJSON(registry Registry) []byte {
+	jsonRegistry, err := json.Marshal(registry)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return jsonRegistry
+}
+
+// Convert a JSON from bytes to the Registry struct format
+func ConvertJSONToRegistry(jsonRegistry []byte, registry Registry) {
+	if err := json.Unmarshal(jsonRegistry, &registry); err != nil {
+		log.Fatal(err)
+	}
+}
```

Comment:
I also do not see why not make this a constructor altogether:
```
func NewRegistryFromJSON(json []byte) *Registry{
    registry := new(Registry)

   if err := json.Unmarshal...

   return registry
}
```

Reply:
Thank you for the explanation. I've modified this. 
I prefer to not add a constructor for the registry. It will be called from the main.go where the service will start working.